api/tests/acceptance/application/attachments_test.js

```diff
@@ -0,0 +1,161 @@
+          url: '/api/attachments',
+          payload: invalidPayload,
+          headers: generateAuthorizationHeader(editorUser),
+        });
+
+        // then
+        expect(response.statusCode).toBe(400);
+      });
+    });
+
+    it('should respond with status 201 and created attachment', async () => {
+      // given
+      databaseBuilder.factory.buildLocalizedChallenge({
+        id: 'challenge123',
+        challengeId: 'challenge123',
+        locale: 'fr',
+      });
+      databaseBuilder.factory.buildLocalizedChallenge({
+        id: validPayload.data.attributes['localized-challenge-id'],
+        challengeId: 'challenge123',
+        locale: 'es',
+      });
+      await databaseBuilder.commit();
+      const airtableAttachment = airtableBuilder.factory.buildAttachment({
+        id: 'airtableAttachmentId',
+        type: validPayload.data.attributes.type,
+        url: validPayload.data.attributes.url,
+        size: validPayload.data.attributes.size,
+        mimeType: validPayload.data.attributes['mime-type'],
+        filename: validPayload.data.attributes.filename,
+        challengeId: 'challenge123',
+        localizedChallengeId: validPayload.data.attributes['localized-challenge-id'],
+      });
+      const airtablePostAttachmentScope = nock('https://api.airtable.com')
+        .post('/v0/airtableBaseValue/Attachments/', {
+          records: [{
+            fields: {
+              'url': validPayload.data.attributes.url,
+              'size': validPayload.data.attributes.size,
+              'type': validPayload.data.attributes.type,
+              'mimeType': validPayload.data.attributes['mime-type'],
+              'filename': validPayload.data.attributes.filename,
+              'challengeId': [validPayload.data.relationships.challenge.data.id],
+              'localizedChallengeId': validPayload.data.attributes['localized-challenge-id'],
+            },
+          }],
+        })
+        .query({})
+        .matchHeader('authorization', 'Bearer airtableApiKeyValue')
+        .reply(200, { records: [airtableAttachment] });
+      const server = await createServer();
+
+      // when
+      const response = await server.inject({
+        method: 'POST',
+        payload: validPayload,
+        url: '/api/attachments',
+        headers: generateAuthorizationHeader(editorUser),
+      });
+
+      // then
+      expect(response.statusCode).toBe(201);
+      expect(response.result).toEqual({
+        data: {
+          type: 'attachments',
+          id: 'airtableAttachmentId',
+          attributes: {
+            'url': validPayload.data.attributes.url,
+            'size': validPayload.data.attributes.size,
+            'type': validPayload.data.attributes.type,
+            'mime-type': validPayload.data.attributes['mime-type'],
+            'filename': validPayload.data.attributes.filename,
+            'localized-challenge-id': validPayload.data.attributes['localized-challenge-id'],
+            alt: null,
```

Comment:
```suggestion
'alt': null
```
Histoire d'être raccord avec les autres attributs

Reply:
oki